terraform-beyond-layer0/example-1/modules/guestbook_service/main.tf

```diff
@@ -1,9 +1,5 @@
-#
```

Comment:
Do we have any walkthroughs on how to write a good terraform module for your service? 
When we first started using terraform with our services, we would just add a `layer0.tf` in the same directory as the service, kind of like a `Dockerfile`. Now we know that "releasing" it as a terraform module is a much better option. 

I assume most people would make the same mistake as us at first. I think it would be a good idea to steer them in the right direction here.